fhirclient/client.py

```diff
@@ -30,64 +30,66 @@ class FHIRClient(object):
         - `scope`: Space-separated list of scopes to request, if other than default
         - `launch_token`: The launch token
     """
-    
+
     def __init__(self, settings=None, state=None, save_func=lambda x:x):
         self.app_id = None
         self.app_secret = None
         """ The app-id for the app this client is used in. """
-        
+
         self.server = None
         self.scope = scope_default
         self.redirect = None
         """ The redirect-uri that will be used to redirect after authorization. """
-        
+
         self.launch_token = None
         """ The token/id provided at launch, if any. """
-        
+
         self.launch_context = None
         """ Context parameters supplied by the server during launch. """
-        
+
         self.wants_patient = False
         """ If true and launched without patient, will add the correct scope
         to indicate that the server should prompt for a patient after login. """
-        
+
         self.patient_id = None
         self._patient = None
 
         self.jwt_token = None
         """ If present, is included as part of the request to authenticate
         with a backend system through a client_assertion parameter
         """
-        
+
+        self.settings = settings
+
```

Comment:
Would a comment make sense here, to let users know what is expected to be passed?

Reply:
I added a section to the readme with these details, please look at that and let me know if it's sufficient.